apps/designer/app/designer/features/style-panel/shared/get-final-value.ts

```diff
@@ -1,6 +1,6 @@
 import type { Style, StyleValue, StyleProperty } from "@webstudio-is/css-data";
 import { toValue } from "@webstudio-is/css-engine";
-import type { InheritedStyle } from "./get-inherited-style";
+import type { InheritedStyle } from "./style-source";
```

Comment:
I have a feeling that  `getFinalValue` is wrong. If we already injected `browserStyles` into currentStyle at `useStyleData` why we didn't do that there too and use some special `getFinalValue` ?

Logic like 
```
  if (currentValue?.type !== "unit" && isNumericString(toValue(currentValue))) {
    return {
      value: Number(toValue(currentValue)),
      type: "unit",
      unit: "number",
    };
  }
  if (
    currentValue &&
    currentValue.type === "unit" &&
    isValid(
      property,
      toValue({ ...currentValue, value: Math.abs(currentValue.value) })
    ) === false
  ) {
    return { value: String(currentValue.value), type: "invalid" };
  }
```
also looks useless.

Why I'm writing this here, that without inheritedStyle looks like everything is needed is 
`currentStyle` and `getStyleSource`

# MAIN IDEA
Why not just return `getStyleSource: property => source` at `useStyleDate`
Having that we don't use other styles